consensus-types/types/block.go

```diff
@@ -58,21 +58,21 @@ func NewBeaconBlockWithVersion(
 	parentBlockRoot common.Root,
 	forkVersion uint32,
 ) (*BeaconBlock, error) {
-	if forkVersion == version.Deneb {
+	switch forkVersion {
+	case version.Deneb, version.Deneb1:
```

Comment:
@calbera: how to we make sure we have made this change in every place we have to make it?

Reply:
yes. Made all the changes so we don't break. However there is more refactoring necessary so that version can be used correctly. Including getting rid of the `.Version()` function on many types.